src/core/layout/scrollable-list/scrollable-list-handlers.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @module core/layout/scrollable-list
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0 Apache-2.0
+ */
+/* eslint-disable no-console */
+
+import fp from "../../../../lib/lodash/fp/fp.js";
+
+const GRID_NAME = "grid";
+
+const handleIfVisible = (gelButton, scene) => {
+    const panel = gelButton.rexContainer.parent.getTopmostSizer();
+    const safeArea = scene.layout.getSafeArea({}, false);
+    const height = scene.scale.displaySize.height;
+    const topY = height / 2 + safeArea.y + panel.space.top;
+    const bottomY = topY + panel.innerHeight;
+    const mouseY = scene.input.y;
+    if (mouseY >= topY && mouseY <= bottomY) onClickPlaceholder(gelButton);
+};
+
+const onClickPlaceholder = gelButton => console.log(`Clicked ${gelButton.config.id}`);
```

Comment:
is this in some requirements? (console logging)

Reply:
I've just refactored this out- needed a placeholder click handler while working on it.

Reply:
a163e18